lib/src/text-input/TextInput.stories.tsx

```diff
@@ -457,6 +457,7 @@ const AutosuggestListbox = () => {
               optional
               placeholder="Choose an option"
               size="fillParent"
+              readOnly
```

Comment:
This test case should not have readOnly.

Reply:
Oops! My fault, I removed it.